### ISSUE TITLE
Feat: Connect with ADB Wifi

### ISSUE THREAD

**Issue by luisbytes** [ionic-team/vscode-extension#136]
I would like to connect my Android device via Wi-Fi to ADB from the extension.

Could you generate the QR code to be scanned from the device

**Comment by dtarnawsky** [comment]
If you have Wifi debugging turned on for your Android Device then then extension will automatically pick this up. 

You will have had to have connected the device at least once via Android Studio.

I am assuming you would like to have it paid to your machine without having to do this in Android Studio once?

**Comment by luisbytes** [comment]
Yes, until now I always have to open Android Studio to connect it with Wifi QR. It would be interesting if the QR was displayed from the extension and the device was connected.
